redis-cloud-aws-ts/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: redis-cloud-aws-ts
```

Comment:
To me, this already matches our convention because the most important provider is Redis Cloud. The fact that we're peering with AWS is secondary. I read it like it's "Redis Cloud, peered with AWS, in TypeScript". We might also have something like "Redis Cloud, peered with Google Cloud, in Python".

@scottslowe What do you think?

Reply:
I'm inclined to agree with @jkodroff here with regard to the ordering of the components. I think Redis Cloud is the primary "cloud service" we're highlighting in this example. That said, I'd probably go with @desteves and use `rediscloud` instead of `redis-cloud`.

Reply:
That works for me. Will fix.